### PR TITLE
Adding java agent for heap info, thread dump logging @ JVM shutdown

### PR DESCRIPTION
**Pull request by jabubake**
Java agent for heap info, thread dump logging @ JVM shutdown.

### CODE REVIEW

**Comment by lesv** [comment]
Travis doesn't fully appreciate your work.

File: monitoring/shutdown-javaagent/pom.xml [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+  </description>
+  <properties>
+    <project.agentjars.dir>${project.parent.build.directory}/agents</project.agentjars.dir>
+    <project.agent.lib.dir>${project.agentjars.dir}/${project.artifactId}/lib/</project.agent.lib.dir>
+  </properties>
+  <dependencies>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-logging</artifactId>
+      <version>0.8.0-beta</version>
```
Comment (lesv) [inline monitoring/shutdown-javaagent/pom.xml]: 0.8.1

**Comment by jabubake** [comment]
Closing this pull request to fix.

File: monitoring/shutdown-javaagent/pom.xml [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+                  <destFileName>shutdown-javaagent.jar</destFileName>
+                </artifactItem>
+              </artifactItems>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+  </build>
+</project>
```
Comment (lesv) [inline monitoring/shutdown-javaagent/pom.xml]: You might wish to add some of the code review tools from java-repo-tools.  (CheckStyle and more, and ErrorProne)

**Pull request closed without merging**